agent/src/main/java/org/openremote/agent/protocol/tradfri/payload/AuthenticateRequest.java

```diff
@@ -0,0 +1,45 @@
+package org.openremote.agent.protocol.tradfri.payload;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.openremote.agent.protocol.tradfri.util.ApiCode;
+
+/**
+ * The class that contains the payload for a request to authenticate to the IKEA TRÅDFRI gateway
+ * @author Stijn Groenen
+ * @version 1.0.0
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class AuthenticateRequest {
```

Comment:
A class to contain a single `string` is a bit overkill

Reply:
This class represents the payload of a request to authenticate a new client to the IKEA TRÅDFRI gateway. This class is used by an object mapper to convert the property in this class to a valid JSON body for the authentication request.

Reply:
Guessed it was to help `jackson` with serialisation....there are other ways to do this but this is quick and easy